src/udev/udevadm-control.c

```diff
@@ -73,10 +73,6 @@ int control_main(int argc, char *argv[], void *userdata) {
                 {}
         };
 
-        r = must_be_root();
-        if (r < 0)
-                return r;
```

Comment:
I think the log message shown by `must_be_root()` is simple and useful for users.
So, I prefer to keep that, but in the later stage. Now we parse and simultaneously process based on the parse result. How about to change that first parse, then operate some non-priv operations, and then `must_be_root()`?